### DISCUSSION TITLE
How do you remove x,y direction labels off of polefigures?

### DISCUSSION THREAD

**Discussion by Juranamou** [mtex-toolbox/mtex#1266] [Ask Anything] [answered]
Whenever I make an ODF there is a pre-programed (x,y) label, along with the label for the crystallographic axis. How can you go about removing these labels?

**Comment by zmichels** [comment, marked as answer]
open the file “mtex_settings.m”

Find the lines that read:
% you can uncomment the following line to disable the annotations
pfAnnotations = @(varargin) [];
setMTEXpref('pfAnnotations',pfAnnotations);

In my file it is at line #53. If you uncomment the line (as I have above), then plots will not include the direction labels/annotations.

Cheers,
Zach

**Reply by Juranamou** [reply]
Thanks so much Zach!
